dashboard.csss-ci.com/server/api/models/Officers_model.php

```diff
@@ -5,12 +5,27 @@
 
 class Officers_model extends CI_Model {
 
+    /**
+     * Data table for class manipulation
+     */
     protected $table = "officers";
 
+    /**
+     * Inserts data in the officers table
+     *
+     * @param Array $data - the data to be inserted into the database
+     * @return Boolean indicating success of insertion
+     */
     public function save($data) {
         return $this->db->insert($this->table, $data);
     }
 
+    /**
+     * Finds and returns data and rows with desired association
+     *
+     * @param Array or String $where - Associative array or custom string
```

Comment:
please describe what the associative array or custom string is (ex. is it going to be inserted?)